alerts/openshift-virtualization-operator/KubeVirtHyperconvergedClusterOperatorPodSchedulingMightBeImbalanced.md

```diff
@@ -0,0 +1,21 @@
+# KubeVirtHyperconvergedClusterOperatorPodSchedulingMightBeImbalanced 
+
+## Meaning 
+
+This alert fires when a node contains more than the maximum configured
+number of images. The problem happens because the list of images on a node
+is shortened to that configured number, 50 by default.
+
+## Impact
+
+Pod scheduling might be imbalanced across nodes.
+
+## Mitigation
+
+As a workaround, you can increase the image limit by editing the
```

Comment:
As a workaround, you can increase the image limit by editing the
`KubeletConfig` object or you can disable the image limit and set
the value of `nodeStatusMaxImages` to `-1`.

Reply:
updated